packages/debugger/README.md

```diff
@@ -0,0 +1,28 @@
+# @frames/debugger
+
+Debug your Frames built using [frames.js](https://framesjs.org) with ease.
+
+## Usage
+
+### Using npx or alternatives
+
+```sh
+$ npx @frames/debugger@latest
+$ pnpm dlx @frames/debugger@latest
+```
+
+### Installing globally
+
+```sh
+# usage with npm
+$ npm install -g @frames/debugger
```

Comment:
we don't have this handle. @frames.js/debugger

Reply:
would it be possible to add this to the main library somehow? a bit like next.js global install

Reply:
we don't have that handle but is also used in eslint config and typescript config package. Should I also fix it there?

Reply:
I changed the handle to @frames.js but kept it as is in eslint and ts config packages as those are private.

Reply:
Also another argument against moving that to main package is that it would affect the version of the package even if there would be just a bug fix in debugger UI for example. I think it should be separated.